acceptance/bundle/paths/fallback_metric/output.txt

```diff
@@ -23,11 +23,11 @@ Deployment complete!
     "value": false
   },
   {
-    "key": "skip_artifact_cleanup",
```

Comment:
The order is changed because now we set wheel wrapper earlier then we do artifacts cleanup